client/src/pages/Search.js

```diff
@@ -502,25 +504,35 @@ export const Tasks = ({
     pageNum: queryParamsUnchanged ? pageNum : 0,
     pageSize: queryParams.pageSize || DEFAULT_PAGESIZE
   })
-  const { loading, error, data } = API.useApiQuery(GQL_GET_TASK_LIST, {
-    taskQuery
-  })
+  const { loading, error, data } = API.useApiQuery(
+    customQuery || GQL_GET_TASK_LIST,
+    {
+      taskQuery
+    }
+  )
   const { done, result } = useBoilerplate({
     loading,
     error,
     pageDispatchers
   })
-  // Update the total count
-  const totalCount = done ? null : data?.taskList?.totalCount
+  // Update the total count, filter first if available
+  const totalCount = done
+    ? null
+    : filterTasks
+      ? filterTasks(data).length
```

Comment:
That was my intention. I don't understand the where the error is. I want to set total count as the number of tasks with pending assessments. if there is a filter, it will filter and then set total count on the filtered items.

Reply:
So you mean that filter acts on both tables ( all responsible tasks - tasks with pending assessments). But we don't give filter props to first table. I didn't test to create this problem as well

Reply:
No, I mean the current pagination page of the pending assessments tasks table.

Reply:
Yes, i understand it now. Needs a fix